src/compiler/Parser/StringParser/StringParser.ts

```diff
@@ -16,7 +16,7 @@ export type ParseRuleReturn =
 
 export type SpecificParseRule<T extends ParseRule> = T;
 
-export abstract class StringParser {
+export class StringParser {
```

Comment:
I removed `abstract` because some tests use `StringParser` as a concrete class and it's (seemingly) not abstract in upstream.